src/components/wallet/UserTransactionsTable.tsx

```diff
@@ -0,0 +1,98 @@
+import React from 'react'
+import { shortSymbol } from '../../helpers/tokenHelpers'
+import { txParty } from '../../helpers/transactionHelpers'
+import { Typography, Table } from 'antd'
+const { Text } = Typography
+
+export type UserTransactionTypes = {
```

Comment:
Let's extract this `type UserTransactionTypes` into `wallet/types` or somewhere, since it has been already used in `UserTransactionsTable`, `transactionHelpers` and might be shared to other places , too.